app/views/accessibility.njk

```diff
@@ -62,16 +62,17 @@
       </p>
       <p class="govuk-body">
         This website is fully compliant with the 
-        <a class="govuk-link" href="https://www.w3.org/TR/WCAG21">Web Content Accessibility Guidelines version 2.2</a> AA standard due to the non-compliances listed below.
+        <a class="govuk-link" href="https://www.w3.org/TR/WCAG21">Web Content Accessibility Guidelines version 2.2</a> AA standard.
```

Comment:
Can you update the link to: https://www.w3.org/TR/WCAG22

Reply:
done thanks